AdblockPlusSafari/NSString+MarkdownRenderer.m

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of Adblock Plus <https://adblockplus.org/>,
+ * Copyright (C) 2006-2015 Eyeo GmbH
+ *
+ * Adblock Plus is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * Adblock Plus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Adblock Plus.  If not, see <http://www.gnu.org/licenses/&gt.
+ */
+
+#import "NSString+MarkdownRenderer.h"
+@import UIKit;
+
+@implementation NSString (MarkdownRenderer)
```

Comment:
I'm fine with going with this approach for now (even though we normally use HTML markup, such as `<strong>` for this purpose so we can reuse translated strings across projects, but we can change that later), but it isn't really a Markdown renderer, it's actually a renderer that can use arbitrary marker chars and use an arbitrary font for text marked like that. So I think it should have a different name. How about something like `TextRenderer`?